package.json

```diff
@@ -27,13 +28,13 @@
     "@googlemaps/markerclusterer": "^2.5.3",
     "@types/google.maps": "^3.58.1",
     "@types/leaflet-routing-machine": "^3.2.8",
-    "@types/leaflet": "^1.9.16",
+    "@types/leaflet": "^1.9.17",
```

Comment:
Why did you update the types of Leaflet? Is there a new version available of the scripts?

Reply:
@rugoncalves No, we did not. We have Leaflet v1.9.4. on the module

Reply:
Thank you for reverting the types. Like said, this should be updated together with the library, when that happens.